src/Http/Role/DeleteConstraintRequestHandler.php

```diff
@@ -26,7 +26,7 @@ public function __construct(CommandBus $commandBus)
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
         $routeParameters = new RouteParameters($request);
-        $roleId = new UUID($routeParameters->get('id'));
+        $roleId = new UUID($routeParameters->getRoleId());
```

Comment:
I think there is one other place where `getRoleId` can be used, inside `UpdateRoleRequestHandler`